nss-day-cohort-51/nss-day-cohort-51.github.io
# Ss navbar fix

# Description

## Type of change


# Testing Instructions
Fetch branch and serve. View in small screen less than 480px. Click on hamburger menu and choose a link. Menu should collapse after clicking. I added attributes to navbar divs.

Also, when I was testing the nav scroll I noticed a need for reduced paddings and margins on small screen which was done with media queries on DevList & TechStack. I also removed my commented out code that is no longer needed.

# Checklist:
